interview-questions.md

```diff
@@ -8,50 +8,50 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-Your answer:
+Your answer:    Conditional statements are if else statements that require a specific condition to be met to produce a specific outcome.
 
-Researched answer:
+Researched answer:  Statements in computer programming languages that run specific courses of action if a certain criteria is met.
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer:    Git is a version control software. Github is the cloud server version of a version control software.
 
-Researched answer:
+Researched answer:  Git is a open source version control software while Github is a web app service that hosts the majority of repositories with collab services and tools.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer:    && and ||
```

Comment:
When answering interview questions, you should expand using definitions and examples when possible. One-word or two-word answers in an interview setting will not show your understanding and knowledge, and will likely come off as uninterested.